### DISCUSSION TITLE
incorrect repo language detection even with gitattributes

### DISCUSSION THREAD

**Discussion by Chanoian** [community/community#22322] [New to GitHub]
<p>This is my repo </p>
<p><a href="https://github.com/Chanoian/SeleniumAutomation" rel="nofollow noopener">https://github.com/Chanoian/SeleniumAutomation</a></p>
<p>It shows that the programming language is html because of a log.html file.</p>
<p>I have created .gitattributes file </p>
<p><a href="https://github.com/Chanoian/SeleniumAutomation/blob/master/.gitattributes" rel="nofollow noopener">https://github.com/Chanoian/SeleniumAutomation/blob/master/.gitattributes</a></p>
<p>but still the same issue </p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/chanoian">@chanoian</a>,</p>
<p>Thanks for being here! GitHub determines language statistics based on the total size of the files that belong to each language.</p>
<p>We use the Linguist open source library to analyze source code files. Linguist supports a number of different custom overrides strategies for language definitions and vendored paths. These are commonly used for documentation, vendor files, or generated files. You can read about how to use overrides in the Linguist documentation:</p>
<p><a href="https://github.com/github/linguist#troubleshooting" rel="nofollow noopener">https://github.com/github/linguist#troubleshooting</a></p>
<p>I hope this helps.</p>
